rd_ui/app/views/users/show.html

```diff
@@ -24,9 +24,22 @@ <h3 class="p-l-5">{{user.name}}</h3>
     </p>
     <div ng-if="currentUser.isAdmin">
       <hr/>
-      <button class="btn btn-default" ng-click="sendPasswordReset()" ng-disabled="disablePasswordResetButton">Send
-        Password Reset Email
-      </button>
+      <div class="form-group">
+        <button class="btn btn-default" ng-click="sendPasswordReset()" ng-disabled="disablePasswordResetButton">Send
+          Password Reset Email
+        </button>
+      </div>
+
+      <div ng-if="passwordResetLink && !disablePasswordResetButton" class="alert alert-success">
+        <p>
```

Comment:
No. We show one message if there is no mail server configured and the other otherwise. But I just noticed that you have the ng-if backwards: it should be `clientConfig.mailSettingsMissing` for your message and `! clientConfig.mailSettingsMissing` for the original one.